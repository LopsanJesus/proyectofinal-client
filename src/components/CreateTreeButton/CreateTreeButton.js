import React from "react";

import Card from "react-bootstrap/Card";
import { PlusCircle } from "react-bootstrap-icons";

import "./CreateTreeButton.scss";
import { useTranslation } from "react-i18next";
import { Link } from "react-router-dom";

const CreateTreeButton = () => {
  const { t } = useTranslation();

  return (
    <Link to="/create-tree">
      <Card className="createTreeButton">
        <Card.Body>
          <PlusCircle size={30} className="plusIcon" />
          <Card.Title>{t('treeList.addTree')}</Card.Title>
        </Card.Body>
      </Card>
    </Link>
  );
};

export default CreateTreeButton;
