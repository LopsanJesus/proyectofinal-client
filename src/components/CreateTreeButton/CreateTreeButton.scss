.createTreeButton {
  min-width: 300px;
  margin-right: 20px;
  margin-bottom: 20px;

  .plusIcon {
    margin-bottom: 8px;
  }

  &:hover {
    cursor: pointer;
    border: 1px solid black;
  }
}
