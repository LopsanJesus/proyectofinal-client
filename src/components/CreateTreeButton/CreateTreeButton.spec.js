import React from 'react';
import { shallow } from 'enzyme';
import CreateTreeButton from './CreateTreeButton';

describe('CreateTreeButton', () => {
  test('matches snapshot', () => {
    const wrapper = shallow(<CreateTreeButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
