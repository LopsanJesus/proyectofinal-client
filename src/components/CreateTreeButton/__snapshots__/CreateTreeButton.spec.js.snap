// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateTreeButton matches snapshot 1`] = `ShallowWrapper {}`;
